proposals/jib-cli-surface.md

```diff
@@ -0,0 +1,60 @@
+# Jib CLI Surface Specification
+
+Specification for a command structures exposed to users of the jib-cli
+
+## Usage
+`jib COMMAND [OPTIONS]`
+
+## Commands
+
+The jib cli exposes the following top level commands, mirroring what is available in the plugins
+```
+build        build to registry
+dockerBuild  build to docker daemon
+buildTar     build to a tar file
+```
+
+## Options
+
+### Required
+```
+-t  --target <image-ref>  a target image reference
+```
+
+### Optional
+```
```

Comment:
This is just a thought and also very much optional: How about having  a separate section for "general purpose flags" such as `help`, `version` and `verbosity`?